components/navigation/topMenu.tsx

```diff
@@ -0,0 +1,23 @@
+import { Icon } from '@mui/material';
+import { FlatLogoIcon } from '../common/flatLogoIcon';
+import HandymanIcon from '@mui/icons-material/Handyman';
+
+export const TopMenu = () => {
+  return (
+    <div className="bg-primary-dark h-16 text-light relative p-4">
+      <div className="h-full float-left flex w-1/4 ">
+        <div className="h-full flex align-middle place-items-center justify-center">
+          <div className="h-full float-left mr-2">
+            <FlatLogoIcon />
+          </div>
+          <h1 className=" float-right text-xl">Athena</h1>
+        </div>
+      </div>
+      <div className="h-full float-right flex align-middle place-items-center justify-center w-10">
+        <HandymanIcon className="w-full " color="inherit" />
+      </div>
+    </div>
+  );
+};
+
+export default TopMenu;
```

Comment:
Does this need doc comments? It is pretty obvious what it is from the code itself.

Reply:
Once again I would direct you to my comment in the description of the PR in which I say that this is a placeholder, Chen is working on the actual component that will be implemented eventually, so this placeholder probably does not need to be documented in any substantial manner